src/util_test.ts

```diff
@@ -86,7 +86,7 @@ describe('isModifiedArray', () => {
     diff: [[], []],
     result: false
   }, {
-    diff: [[' ', 'unchangedElement'], ['+', 'addedElement']],
```

Comment:
Is the intent here that unchanged array elements just don't show?

Reply:
That's correct; only elements removed or deleted should be shown in the diff view (open to suggestions on this though)

Reply:
As long as it's paired with an array index I think it's fine, just making sure I understand.